common/reviews/api/node-core-library.api.ts

```diff
@@ -7,14 +7,20 @@ class Colors {
   // (undocumented)
   static blackBackground(text: string | IColorableSequence): IColorableSequence;
   // (undocumented)
+  static blink(text: string | IColorableSequence): IColorableSequence;
```

Comment:
Ermmm.... is `blink` professional?

Reply:
I'd rather be unopinionated about that. It's one of the supported formats, so I'd rather include it.